client/src/entrypoints/admin/bulk-actions.js

```diff
@@ -32,30 +32,31 @@ function SelectBulkActionsCheckboxes(e) {
   if (e.target.checked) checkedState.checkedPages.add(+e.target.dataset.pageId);
   else {
     /* unchecks `Select all` checkbox as soon as one page is unchecked */
-    document.querySelector(`.${BULK_ACTION_SELECT_ALL_CHECKBOX_TH} input`).checked = false;
+    document.querySelectorAll(`.${BULK_ACTION_SELECT_ALL_CHECKBOX_TH} input`).forEach(_el => {
+      const el = _el;
```

Comment:
Is this necessary?

Reply:
A linting issue shows up if I try to modify the property of the function parameter